cassandra/metadata.py

```diff
@@ -416,8 +420,8 @@ def _build_table_metadata(self, keyspace_metadata, row, col_rows, trigger_rows):
                 if value_alias_rows:  # CASSANDRA-8487
                     validator = types.lookup_casstype(value_alias_rows[0].get('validator'))
 
-            col = ColumnMetadata(table_meta, value_alias, validator)
-            if value_alias:  # CASSANDRA-8487
+            if value_alias and validator:  # CASSANDRA-8487
```

Comment:
Seems like ``validator`` could be uninitialized here, although that was also true in the old code.